scripts/grype_report_parser.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type VulnerabilityScannerResponse struct {
+	Matches                json.RawMessage `json:"matches"`
+	RelatedVulnerabilities json.RawMessage `json:"relatedVulnerabilities"`
+	MatchDetails           json.RawMessage `json:"matchDetails"`
+	Artifact               json.RawMessage `json:"artifact"`
+}
+
+type MatchResponse struct {
+	Vulnerabilities VulnerabilityReport `json:"vulnerability"`
+}
+
+type FixVersionsResponse struct {
+	Versions []string `json:"versions"`
+	State    string   `json:"state"`
+}
+
+type VulnerabilityReport struct {
+	Id          string              `json: id`
+	Severity    string              `json:"severity"`
+	Namespace   string              `json:"namespace"`
+	Description string              `json:"description"`
+	FixVersions FixVersionsResponse `json:"fix"`
+}
+
+// Filters vulnerabilites based on the severity levels set in severityTypeSet
```

Comment:
Comments should be of the form - `<name> <comment>`
For example: `filterVulnerabilityReportMatches filters vulnerabilities based on the severity levels set in severityTypeSet`
Read more [here](https://go.dev/doc/comment)

Reply:
```suggestion
// Filters vulnerabilities based on the severity levels set in severityTypeSet
```
[typo]

Reply:
Thanks for pointing this out. I've fixed the comments based on the suggested format. Fixed the typo as well.